api/topic_private.go

```diff
@@ -117,3 +117,35 @@ func (api *API) getSubtopicsPrivateByID(ctx context.Context, id string, logdata
 	}
 	log.Info(ctx, "request successful", logdata) // NOTE: name of function is in logdata
 }
+
+func (api *API) putTopicReleaseDatePrivateHandler(w http.ResponseWriter, req *http.Request) {
+	ctx := req.Context()
+	vars := mux.Vars(req)
+	id := vars["id"]
```

Comment:
(suggestion): I Guess here it would be good to add a validation for the id right ? I mean 

```
if id == "" {
handlerError()
  return
}
```